phpstorm/CheckBuildUpdate.ps1

```diff
@@ -26,6 +26,15 @@ catch {
     }
 }
 
+# Jetbrains can be a little inconsistent in their naming of document pages.
+# So we verify the release specific page actually exist where we think it should.
+$release_url = "https://confluence.jetbrains.com/display/PhpStorm/PhpStorm+$newVersion+Release+Notes"
+$download_release = Invoke-WebRequest -Uri $release_url -UseBasicParsing
+# If not fallback to a documentation page that contains a list of all release note pages.
+if($download_release.RawContent -like '*Page Not Found*') {
```

Comment:
I'd prefer not relying on some certain text to be written by 3rd party software when we have defined error codes.
The Invoke-WebRequest throws an exception when the response is a 4xx or 5xx status and Page Not Found will most certainly always be a 404 error. 

````posh
try {
    $release_url = "https://confluence.jetbrains.com/display/PhpStorm/PhpStorm+$newVersion+Release+Notes"
    $response = Invoke-WebRequest -Uri $release_url -method head
} catch {
    # Check for 404 status code and ignore other status codes that might be temporary only (i.e. 5xx codes)
    if ($_.Exception.Response.StatusCode -eq 404) {
        $release_url = "https://confluence.jetbrains.com/display/PhpStorm/PhpStorm+Release+Notes"
    }
}
````